yapapi/strategy/__init__.py

```diff
@@ -0,0 +1,19 @@
+from typing import Union
+
+from .base import (
+    DEBIT_NOTE_INTERVAL_GRACE_PERIOD,
+    MarketStrategy,
+    PropValueRange,
+    PROP_DEBIT_NOTE_ACCEPTANCE_TIMEOUT,
+    PROP_DEBIT_NOTE_INTERVAL_SEC,
+    PROP_PAYMENT_TIMEOUT_SEC,
+    SCORE_NEUTRAL,
+    SCORE_REJECTED,
+    SCORE_TRUSTED,
+)
+from .decrease_score_unconfirmed import DecreaseScoreForUnconfirmedAgreement
+from .dummy import DummyMS
+from .least_expensive import LeastExpensiveLinearPayuMS
+from .wrapping_strategy import WrappingMarketStrategy
+
+MarketStrategyType = Union[MarketStrategy, WrappingMarketStrategy]
```

Comment:
This name suggests a type (e.g. `MyMarketStrategy`), not an instance (`MyMarketStrategy()`).
Check e.g. `EventType`, `JobEventType` etc in `yapapi.events`.

--> proposed improvement is in the main comment